### ISSUE TITLE
Pinned Streams: Should be easily accessible.

### ISSUE THREAD

**Issue by ishammahajan** [zulip/zulip-mobile#3419]
Pinned streams exist for one purpose only, to be easily accessible. The current implementation, pinning them on the top, makes it unwieldy for someone with larger phones (a lot of them are large these days, I don't get the craze at all) to reach to the top to click on the stream. This becomes an even bigger problem with people who have relatively smaller hands. This defeats the purpose of pinned streams.

A solution to this problem could be a floating window similar to the one which comes up when autocompleting a stream or a mention in `ChatScreen` at the bottom of the streams tab which lists the pinned streams.

**Comment by pulkonet** [comment]
Another solution would be to rethink the whole UI of the app like the One UI by Samsung, but that would be a much bigger task. A floating window would be absurd IMO if you have more than 5 pinned streams.
We can make mockups and find out.

**Comment by ishammahajan** [comment]
A possible solution to that would be a floating window, but with a maximum line and scrolling if it does cross the limit. I don't think a redesign is required though, the current UI is rather good! 🙂

**Comment by pulkonet** [comment]
Yes, the current UI is great; and I mentioned the UI overhaul because taller displays were in context. One UI is really killing it there.

**Comment by ishammahajan** [comment]
That's interesting, it seems like a very drastic change, but would bring lots of benefits with it. I wonder what the maintainers think about such a change.

cc @gnprice @borisyankov

Top 1/3rd screen is designed to be read-only at launch
![](https://user-images.githubusercontent.com/33200024/54719452-f3d27b00-4b82-11e9-9e5d-49e351c46b14.png)
